deps/buildutils.jl

```diff
@@ -0,0 +1,102 @@
+# Included from build.jl and ../test/test_build.jl
+
+using Compat, VersionParsing
+import Conda, Compat.Libdl
+
+# Fix the environment for running `python`, and setts IO encoding to UTF-8.
+# If cmd is the Conda python, then additionally removes all PYTHON* and
+# CONDA* environment variables.
+function pythonenv(cmd::Cmd)
+    env = copy(ENV)
+    if dirname(cmd.exec[1]) == abspath(Conda.PYTHONDIR)
+        pythonvars = String[]
+        for var in keys(env)
```

Comment:
shouldn't we just call `Conda._set_conda_env`?

Reply:
Yeah, this may be required for Conda with Python >= 3.7.  https://github.com/JuliaPy/Conda.jl/issues/125#issuecomment-423755365

It's probably better to define `Conda.run` which is like `Base.run` but with appropriate environment variables. I think you'll need it in IJulia as well to let Python running Jupyter server to load appropriate DLL in Windows.  Using "private" function starting with underscore is a bit awkward.